src/static/images/icons/notification.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <path d="M15.2,4.2c0,0,0.2-1.3,0.8-2H4.9c0,0-3.8,0-4.1,4.1V19c0,0,0.2,3.6,4.1,3.8h12.5c0,0,3.9,0,4.1-4.1V8.8c0,0-1,0.4-2,0.3
+		v9.6c0,0,0,2-2.1,2.1H4.9c0,0-1.9,0.1-2.1-2.1V6.3c0,0,0-2,2.1-2.1H15.2z"/>
+	<circle cx="19.6" cy="4.7" r="3"/>
+</svg>
```

Comment:
I noticed that this icon looked pretty fuzzy on a standard-resolution screen:

![screen shot 2016-11-02 at 11 05 43 am](https://cloud.githubusercontent.com/assets/69633/19941198/6e88751e-a0ec-11e6-9923-8aab3871e361.png)

It looked like it might be because of the amount of decimals being used to define the shape. I went into Illustrator and tried to recreate it while minimizing decimal points, and I think the result is a bit more crisp:

![screen shot 2016-11-02 at 11 06 14 am](https://cloud.githubusercontent.com/assets/69633/19941237/920d2b6a-a0ec-11e6-8898-66cd5198ea3a.png)

It's also just a tad smaller in filesize. Source here:

```html
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
  <path d="M20,18a2,2,0,0,1-2,2H6a2,2,0,0,1-2-2V6A2,2,0,0,1,6,4h9a5,5,0,0,1,.42-2H6A4,4,0,0,0,2,6V18a4,4,0,0,0,4,4H18a4,4,0,0,0,4-4V8.58A5,5,0,0,1,20,9Z"/>
  <circle cx="20" cy="4" r="3"/>
</svg>
```